packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -0,0 +1,248 @@
+                                    {({ field, meta: { touched, error } }: FieldProps<typeof initialValues>) => (
+                                        <React.Fragment>
+                                            <DesktopWrapper>
+                                                <Autocomplete
+                                                    {...field}
+                                                    {...fields[field.name as keyof typeof fields]}
+                                                    data-lpignore='true'
+                                                    autoComplete='off' // prevent chrome autocomplete
+                                                    error={(touched && error) as string}
+                                                    onItemSelection={onItemSelection(field.name, setFieldValue)}
+                                                    data-testid={`dt_${field.name}`}
+                                                />
+                                            </DesktopWrapper>
+                                            <MobileWrapper>
+                                                <SelectNative
+                                                    {...field}
+                                                    {...fields[field.name as keyof typeof fields]}
+                                                    error={(touched && error) as string}
+                                                    use_text
+                                                    should_hide_disabled_options={false}
+                                                    data-testid={`dt_${field.name}`}
+                                                />
+                                            </MobileWrapper>
+                                        </React.Fragment>
+                                    )}
+                                </Field>
+                            </fieldset>
+                            <fieldset
+                                className={classNames(
+                                    'additional-kyc-info-modal__form-field',
+                                    'additional-kyc-info-modal__form-field--info'
+                                )}
+                            >
+                                <Field name='tax_residence'>
+                                    {({ field, meta: { touched, error } }: FieldProps) => (
+                                        <React.Fragment>
+                                            <DesktopWrapper>
+                                                <Autocomplete
+                                                    {...field}
+                                                    {...fields[field.name as keyof typeof fields]}
+                                                    data-lpignore='true'
+                                                    autoComplete='off' // prevent chrome autocomplete
```

Comment:
do we need this comment?

Reply:
I think we can keep it as it doesn't hurt anyways